src/Components/DynamicFields.res

```diff
@@ -390,7 +412,16 @@ let make = (
           }
         | SpecialField(_)
         | InfoElement
-        | None => ""
+        | _ => ""
+        }
+        switch item.field_type {
```

Comment:
This should not be added here, since StateAndCity is something which we are internally using for rendering of Input Fields, it will not be coming from the Backend and will not be there in `requiredFieldsType `